.code-samples.meilisearch.yaml

```
@@ -278,6 +278,14 @@ search_parameter_guide_crop_marker_1: |-
     'attributesToCrop': ['overview'],
     'cropMarker': '[…]'
   })
+search_parameter_guide_matching_strategy_1: |-
+  client.index('movies').search('big fat liar', { 
+    matchingStrategy: 'last'
```

I think it should contains matchingStrategy in quotes like this:
```suggestion
    'matchingStrategy': 'last'
```

---

Yes, it does! Also, there is some extra space a the end of certain lines.